package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "videojs-contrib-hls",
-  "version": "5.12.2",
-  "description": "Play back HLS with video.js, even where it's not natively supported",
+  "name": "@videojs/http-streaming",
+  "version": "0.1.0",
+  "description": "Play back HLS and DASH with video.js, even where it's not natively supported",
   "main": "es5/videojs-contrib-hls.js",
```

Comment:
What do we want the output files to be named? `videojs-http-streaming.js`?

Reply:
Sounds good to me.

Reply:
Makes sense to me